packages/wallet/src/utils/nitro-converter.ts

```diff
@@ -110,7 +110,8 @@ export function convertAddressToBytes32(address: string): string {
   // We pad to 66 = (32*2) + 2('0x')
   return `0x${normalizedAddress.substr(2).padStart(64, "0")}`;
 }
-function convertAllocationToOutcome({ allocation, destination }: { allocation: string[]; destination: string[] }): Outcome {
+
+export function convertAllocationToOutcome({ allocation, destination }: { allocation: string[]; destination: string[] }): Outcome {
```

Comment:
Super long line?

Reply:
Yeah mainly because of the ad-hoc typing used.

Will update the typing here in a separate PR.